docker/mssql-tools/Dockerfile

```diff
@@ -7,4 +7,7 @@ RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/${kubectl
     && chmod +x ./kubectl \
     && mv ./kubectl /usr/local/bin/kubectl
 
+COPY --from=ghcr.io/kastenhq/kopia@sha256:ad987e7062686fb1c2485e7c1a94268708682152d97f2160a4b93aa514d65b7e \
+  /kopia/kopia /usr/local/bin/kopia
+
```

Comment:
Do we really need this? kando internall calls kopia client, so I don't think this binary is required